src/main/java/core/basesyntax/ReadFromFile.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+import core.basesyntax.interfaces.ReadFromFileInt;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class ReadFromFile implements ReadFromFileInt {
```

Comment:
```suggestion
public class FileReaderImpl implements FileReader {
```
we already have FileWriter - let's make class names uniform

Reply:
done